### DISCUSSION TITLE
Special Announcement#3: 与月亮计划的第三次沟通

### DISCUSSION THREAD

**Discussion by N071** [LocalizeLimbusCompany/LocalizeLimbusCompany#178] [公告 | Announcements]
Although most of the users of this project use CN as their primary language, EN will be used here since we're on GitHub.
Please kindly refer to [discussion#165](https://github.com/orgs/LocalizeLimbusCompany/discussions/165), as it provides context of this following-up negotiation.

***This is not finished. Please wait for further information.***

**Comment by N071** [comment]
I requested a Bcc from our Korean language user, who's responsible for sending this e-mail.
We started the conversation by forwarding the past email thread mentioned in [discussion#165](https://github.com/orgs/LocalizeLimbusCompany/discussions/165), and added the following texts:
## Original screenshot
<details>

One of PM's private email addresses is in the cc list; hence it was redacted and will not be shown.
![image](https://github.com/LocalizeLimbusCompany/LocalizeLimbusCompany/assets/128023427/873d96dd-7863-45c1-9bcc-bc5a0fe10321)
![image](https://github.com/LocalizeLimbusCompany/LocalizeLimbusCompany/assets/128023427/5a353ca7-ce19-434c-b2bf-f2789dc2713a)
</details>

## KR Raw
<details>

존경하는 프로젝트문에게:

안녕하십니까.뒤늦게 지난 메일의 답장을 보내게 되여 참으로 죄송합니다.저희는 프로젝트문이 TGS2023의 활동을 끝마친후에 이번의 메일을 통하여 지난 메일로하여 생긴 오해를 풀어낼 계획이였습니다.

처음으로는 지난번의 메일에서 모호하고 적합하지 않은 용어로하여 오해를 이르켜 사과드립니다.그리고 프로젝트문을 연락하기 위해 함부로 Business메일을 사용하여 미안합니다.저희 목적은 공식적인 합동을 체결하려는 것도 아니고,림버스 컴퍼니를 중국 시장으로 인입하려는 것도아니였습니다.저희는 커뮤니티의 일원으로서 중국의 심사제도와 인입 난이도에 대해서 그 누구보다도 더 잘알고 있습니다.

다시한번 저희를 소개하겠습니다.저희는 중국의 LCB 플레이어들로 구성된 단체입니다.저희 성원들은 한국어,일본어,영어를 능숙하게 다룰수 있으며 몇분의 프로그래머들의 도움하에 LCB의 번역 커뮤니티를 건립하였습니다.저희는 중국의 플레이어들에게 중국어 번역 모듈을 완전 무료로 제공하고 그 어떤 방식으로도 이윤을 추구하지 않습니다.

저희가 지난번에 방문을 하려는 것은 합동 체결을 위한 것이 아니고,그저 프로젝트문에게 저희 작품을 보여주려는 기회를 갖고 싶었던 것이였습니다.

저희는 프로젝트문의 작품을 번역하는 “머나먼” 중국의 커뮤니티 조직으로서 프로젝트문의 인정을 갈망합니다.4월 BattlEye도입 시,번역만을 위한 모듈 사용에 대해서는 계정을 정지시키지 않겠다고 승낙하여 대단히 감사합니다!하지만 저희는 프로젝트문과 진일보로 접촉해보고 싶습니다.

저희는 현단계의 프로젝트문이 매우 바쁘게 보내고 있다는것을 알만합니다.그래서 저희도 도움을 청하여 프로젝트문의 스트레스를 증가해서는 안된다는것도 압니다.저희는 취미와 사랑을 동력으로 삼는 조직이기에 상업적으로 합동을 체결하지 않을것입니다.그러나 우리는 프로젝트문과 믿음을 토대로하여 더욱 긴밀한 관계를 맺기를 희망합니다.이런 믿음을 동력으로써 저희 성원들과 프로젝트문의 중국 팬들을 고무격려할수 있었으면 좋겠습니다.

직접 저희의 성과를 보여드릴수 있는 기회가 없어서 안타깝습니다.저희 모듈은 목요일의 업데이트 1-2일내로 동기화되여 새로운 내용의 중국어 번역을 보충합니다.이와 동시에 저희 오픈 소스 프로젝트(https://github.com/LocalizeLimbusCompany/LocalizeLimbusCompany) 와 기타 언어 커뮤니티의 지원자들의 노력 덕분에 저희 모듈은 지금 중국어 번체(https://github.com/SmallYuanSY/LocalizeLimbusCompany) ,러시아어 (https://github.com/Crescent-Corporation/LimbusCompanyBusRUS) 태국어 (https://github.com/1ookilo/LocalizeLimbusCompanyTH) 를 지지 할수도 있게 되였습니다.이에 더불어,LCB의 모든 통지 내용은 발표 1시간내에 정확히 번역되여 저희의 계정에 올리게 됩니다.

저희는 프로젝트문과 직접 소통할수있는 수단이 모자란 중국에 있기에 비록 많이 온정적이고 많은 LCB 팬들의 지지를 받았지만 저희는 아직도 번역 모듈에 대해 장기적으로 독립하고 점차 탐색하는 상태입니다.그리하여 혹시나 가능하다면 저희는 프로젝트문이 저희가 이러한 모듈의 제작에 대한 공식적인 허가를 받을수 있었으면 좋겠습니다.문자나 문서 형식의 허가를 받을수 있으면 저희 의지를 더욱 든든하게 할수도있고 미래에도 장기적이고도 지속적으로 번역을 제공할 용기도 낼수 있게됩니다.

저희 기나긴 해석을 끝까지 읽어 주셔서 대단히 감사합니다.그리고 프로젝트문이 저희 허가와 기대에 관한 부탁을 고려해 주시기를 바랍니다.
</details>

## Original language: CN
<details>

This mail is written with Chinese, then translated into Korean. Here's the original Chinese text.

我们敬爱的月亮计划，您们好。很抱歉自我们上一次沟通以来，我们现在才针对上次并未解决的问题发来回信。我们希望在您完成TGS2023的事务后再发来这封邮件。

首先，请允许我为我们在上次的沟通中不严谨的用语道歉。也很抱歉为了联系到您，我们擅自选用了您的Business邮箱。我们发这封邮件是为了解开一个误会：我们的意图并不是与您签订正式的翻译协议，也并非意图邀请您进入中国市场。作为社区的一部分，我们很清楚中国的审查制度与市场的进入难度。

请让我重新表述我们的身份。我们是一些来自中国的LCB玩家组成的一个集体，我们的成员熟悉韩语、日语与英语，并在几位程序员的辅助之下构建了一个LCB的翻译社区。我们为中国玩家提供中文翻译模组，但我们从不以任何方式盈利，或将我们的模组放在付费门槛之后。

我们之前和您商量的到访，并不是为了与您签订任何形式的协议，仅仅为了争取一个向您展示我们的作品的机会。
作为一个为您的游戏提供翻译，并且身处遥远的中文社区的组织，我们渴望您的认同。此前，您向我们承诺将翻译文件置入白名单，且并不会惩罚使用模组的玩家，已经让我们非常高兴和感激。不过，我们希望同您进行更进一步的接触。

我们知道您的工作繁重，因此我们也明白我们不能索要帮助，因而加重您的压力。我们也明白，作为一个以爱好为动力的社区组织，我们也不会签订任何商业协议。不过，我们仍然希望在我们之间建立更加紧密的信任关系，并以这样的信任为动力继续激励我们的成员和您在中国的粉丝。

我们很少有机会向您亲自展示我们的成果。我们的模组一般在您每周四更新之后的1-2天内同步更新，并带来完整的中文翻译。同时，基于我们的开源项目(https://github.com/LocalizeLimbusCompany/LocalizeLimbusCompany) 和其他语言社区的有志者的努力，我们的模组已经能按照繁体中文(https://github.com/SmallYuanSY/LocalizeLimbusCompany) 、俄语(https://github.com/Crescent-Corporation/LimbusCompanyBusRUS) 和泰语(https://github.com/1ookilo/LocalizeLimbusCompanyTH) 提供。在此之上，自LCB开服以来，每当您发布游戏公告，我们都会在一小时内将其准确地翻译完毕，并转发到我们的社交帐号上。

在与您直接沟通路径较少的中国，虽然我们已经相当稳定，也获得了许多来自LCB的粉丝的支持，但我们对于模组翻译依然长期处于独立、逐渐摸索的状态。因此，如果可以的话，我们希望得到您作为官方对我们制作这样的模组的正式许可。如果我们能得到文字或文书形式的许可，这将使我们更加坚定，使我们有继续长期提供翻译服务的勇气。

非常感谢您愿意听我们解释情况，也希望您能考虑我们对认可的期待和请求。

LocalizeLimbusCompany 敬上
</details>

## English version for your convenience
<details>

Our beloved ProjMoon, Hello. We apologize that since our last communication, we are only now sending a reply to an issue that was not resolved last time. We would like to send this email after you have completed your TGS2023 business.

First of all, let me apologize for the loose language we used in our last communication. We also apologize for using your Business email address in order to reach you. We are sending this email to clear up a misunderstanding: our intention is not to sign a formal translation agreement with you, nor to invite you to enter the Chinese market. As part of the community, we are well aware of the censorship in China and the difficulty of entering the market.

Let me rephrase who we are. We are a collective of LCB players from China who are familiar with Korean, Japanese, and English, and with the help of a few programmers, we have built a community of LCB translators. We provide Chinese translation mods for Chinese players, but we never monetize our modules by any means, or put our modules behind a pay threshold.

The visit we discussed with you earlier was not for the purpose of entering into any kind of business agreement with you, but simply to gain an opportunity to show you our work.
As an organization that provides translations for your games and is located in the distant Chinese community, we are eager for your approval. We have been very pleased and appreciative of your promise to whitelist translations and not penalize players for using modules. However, we would like to engage with you further.

We realize that you have a lot of work to do, and we understand that we cannot add to your stress by asking for more help. We also understand that as a hobby-driven community organization, we will not be entering into any commercial agreements either. However, we still hope to build a stronger relationship of trust between us and use that trust as a motivation to continue to inspire our members and your fans in China.

We rarely have the opportunity to show you our results in person. Our modules are usually synchronized within 1-2 days after your weekly update on Thursdays, bringing full Chinese translations. Meanwhile, based on our open-source project (https://github.com/LocalizeLimbusCompany/LocalizeLimbusCompany) and the efforts of aspiring members of other language communities, our modules have been able to be translated in Traditional Chinese (https://github.com/SmallYuanSY/LocalizeLimbusCompany), Russian (https://github.com/Crescent-Corporation/LimbusCompanyBusRUS) and Thai (https://github.com/1ookilo/LocalizeLimbusCompanyTH). On top of that, whenever you post a game announcement since the start of LCB, we will have it accurately translated and forwarded to our social accounts within an hour.

In China, where there are fewer direct communication paths with you, although we have been quite stable and have gained a lot of support from LCB fans, we are still in a long-term independent and gradual state of figuring out the translations. Therefore, if possible, we would like to get your official permission to make such a mod. If we could get permission in the form of text or paperwork, it would strengthen us and give us the courage to continue providing translation services for a long time.

Thank you very much for your willingness to listen to us explaining the situation, and we hope you will consider our expectations and requests for recognition.
</details>

## Our intention and additional information
### Regarding Discussion 165
The last email in [discussion#165](https://github.com/orgs/LocalizeLimbusCompany/discussions/165) is more or less of a communication disaster. We must clear up misunderstandings this time.

### Expectations
We have abandoned the unrealistic dream of incorporation or direct co-op with ProjMoon a long time ago, even earlier than [the first contact](https://github.com/orgs/LocalizeLimbusCompany/discussions/70).
I know that Bright recently wanted (or should I say "visioned") to ask for additional help or utility instead, e.g., a sneak preview or an early access to texts (which needed to be translated) before each update.

But still, I do not put much hope into this relatively humble expectation. ProjMoon seems to already be pushing itself to the limit, and there's few or none abundancy of time that they can give to us.

I decided to ask for stressless things instead, such as the so called "approval" this time, and hope to close up our relationship with ProjMoon. Rome wasn't built in a day, so does our relationship with ProjMoon.

**This is just the first step, and we hope for a soon and proactive reply from ProjMoon**.

### Reply time and mailbox
It is worth mentioning that PM is more responsive to emails that go to its BUSINESS mailbox (business@projmoon.com). Referring to [discussion#165](https://github.com/orgs/LocalizeLimbusCompany/discussions/165) again, they replied to us just the following day.

Though the things that we are trying to convey might not be 100% "business" related, (which we are aware of and apologized in advance) we still decided to use their business mailbox for better odds, since PM don't always reply every email and has ignored us a couple of times.

**Reply by 1Knightey1** [reply]
Also, for your future information: I "help" with the code of French localization for Limbus (they do not have their releases yet) and offered a hand for Portugese translator of LC and LoR (they are not sure they have the power for something aside from the story tho), so your module has helped for a much bigger amount of translations!
